Sources/LiveViewNative/Views/Layout Containers/Collection Containers/Table.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  Table.swift
+//
+//
+//  Created by Carson Katri on 2/21/23.
+//
+
+#if !os(watchOS)
+import SwiftUI
+
+struct Table<R: CustomRegistry>: View {
+    @ObservedElement private var element: ElementNode
+    private let context: LiveContext<R>
+    @Environment(\.self) private var environment
```

Comment:
What is this used for? It will, I believe, cause the view to be re-rendered every time any environment value changes, so I'd really like to avoid it.